jobbergate-docs/docs/source/elements/apps/cli.md

```diff
@@ -137,3 +137,8 @@ provide the following sub-commands:
 Details for each subcommand can be viewed by passing the `--help` flag to any of them.
 
 Use the `--help` option to explore the CLI and disccover the usage and options for all the subcommands.
+
+
+## Usability
```

Comment:
I think this could be a note to get a little more attention from the users: https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:note

Also, what do you think of citing it is a constraint by a dependency and adding that link for reference?

Reply:
Does this look better?
<img width="760" alt="image" src="https://github.com/omnivector-solutions/jobbergate/assets/8601883/0d5612a6-3770-4b1a-b751-45d2d62410f0">

Reply:
Yes, thanks